src/main/include/subsystems/LimeLightUtil.h

```diff
@@ -52,7 +57,9 @@ namespace t34
 
         void Periodic();
 
-        inline void setTargetMode(TargetMode mode) { m_target_mode = mode; };
+        inline void SetTargetMode(TargetMode mode) { m_target_mode = mode; };
+
+        inline double GetTargetID() { return m_target_id; }
```

Comment:
As this is just an accessor (_not a mutator_) suggest adding `const`:

```suggestion
        inline double GetTargetID() const { return m_target_id; }
```